Week5/joyeeon/Mission/src/apis/api.ts

```diff
@@ -0,0 +1,23 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse, AxiosError } from 'axios';
```

Comment:
현재는 `AxiosRequestConfig` , `AxiosError` 타입을 import했지만 실제 코드에서 사용하지 않고 있습니다. 사용하지 않는 import문은 코드 가독성을 해치고 번들 사이즈에도 영향을 줄 수 있으니, 사용하지 않으면 제거하는 것이 좋을 것 같습니다!

```suggestion
import axios, { type AxiosInstance, type AxiosResponse } from 'axios';
```